app/assets/stylesheets/tailwind_overrides.css

```diff
@@ -17,27 +17,33 @@
   }
 
   body {
-    h1 {
+    h1,
+    .h1 {
       @apply my-4 text-3xl font-bold;
     }
 
-    h2 {
+    h2,
+    .h2 {
```

Comment:
Motivation for this change: https://github.com/davidrunger/david_runger/pull/6572/files#r2019725130

This caused a bug, fixed here: https://github.com/davidrunger/david_runger/pull/6589